recipe/yum_requirements.txt

```diff
@@ -1,6 +1,6 @@
 csh
 imake
-libx11-devel
+libX11-devel
```

Comment:
@ocefpaf,
Ahh, I see. I am vaguely familiar with conda-smithy as I had to re-render the feedstock when I enabled OS X builds. I'll try re-rendering now.